logstash-core/lib/logstash/patches/manticore/trust_strategies.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+
+require 'manticore'
+
+unless defined?(::Manticore::Client::TrustStrategies)
+  module CATrustedFingerprintSupport
```

Comment:
1 of 2: `s/CATrustedFingerprintSupport/TrustStrategySupport/g`

```suggestion
  module TrustStrategySupport
```